### ISSUE TITLE
[QUESTION] Does it support .NET 8 ?

### ISSUE THREAD

**Issue by ClassyCircuit** [litedb-org/LiteDB#2403]
I am considering using LiteDB for my latest project and I would like to know if it supports .NET 8.
Also, does it support latest Entity Framework in combination with .NET 8?

**Comment by gkathire** [comment]
Ive been using it in production for couple of weeks and its works fine with .net core 8

**Comment by JKamsker** [comment]
Yes. LiteDB 5.x (current stable: 5.0.21) targets .NET Standard 2.0 (plus `netstandard1.3` and `net45`), so it runs on .NET 8 as is — just `dotnet add package LiteDB`. The 6.x prereleases additionally ship a native `net8.0` build.

There is no Entity Framework integration, though. LiteDB is a document database with its own API (`LiteDatabase`, `ILiteCollection<T>`, LINQ-style queries via `collection.Query()`), not an EF Core provider. That request is tracked in #1198.

One caveat if you plan to use Native AOT on .NET 8: v5 compiles expressions at runtime and is not AOT-compatible (see #2428).

Closing this as answered — if it doesn't cover your case, comment here and we'll reopen it.
